### PR TITLE
Remove all JavaScript files from the repository

### PR DESCRIPTION
**Pull request by rpominov**
If the main repository will contain only the specification itself it will be easier to maintain and version. We can place the code in accompanying repositories under `fantasyland` organization.

This is not a new idea, @davidchambers already doing this in https://github.com/fantasyland/fantasy-land/pull/284 , this PR goes a bit further though.

Also, it's easier to remove what we would have to update otherwise. Please comment if you have a better course of action in mind.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,35 +5,12 @@
...
-    reduce: 'fantasy-land/reduce',
-    traverse: 'fantasy-land/traverse',
-    chain: 'fantasy-land/chain',
-    chainRec: 'fantasy-land/chainRec',
-    extend: 'fantasy-land/extend',
-    extract: 'fantasy-land/extract',
-    bimap: 'fantasy-land/bimap',
-    promap: 'fantasy-land/promap'
-  };
+  var empty = {};
```
Comment (davidchambers) [inline index.js]: Shall we remove __index.js__ too?

Comment (davidchambers) [reply index.js]: It's fine. [xyz][1] only provides a readme and a shell script (no JavaScript files). npm doesn't mind. :)


[1]: https://github.com/davidchambers/xyz

Comment (rpominov) [reply index.js]: It's gone :)

**Comment by rpominov** [comment]
OK, I'm going to go ahead and merge this.

**Pull request merged**